ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -289,6 +292,17 @@ public Builder applyMainnetNetworkDefaults() {
               "enr:-LK4QKWrXTpV9T78hNG6s8AM6IO4XH9kFT91uZtFg1GcsJ6dKovDOr1jtAAFPnS2lvNltkOGA9k29BUN7lFh_sjuc9QBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpC1MD8qAAAAAP__________gmlkgnY0gmlwhANAdd-Jc2VjcDI1NmsxoQLQa6ai7y9PMN5hpLe5HmiJSlYzMuzP7ZhwRiwHvqNXdoN0Y3CCI4yDdWRwgiOM");
     }
 
+    public Builder applyPraterNetworkDefaults() {
+      return reset()
+          .constants(PRATER.constantsName())
+          .startupTimeoutSeconds(120)
+          .eth1DepositContractAddress("0xff50ed3d0ec03aC01D4C79aAd74928BFF48a7b2b")
+          .eth1DepositContractDeployBlock(4367322)
+          .initialState(
+              "https://github.com/eth2-clients/eth2-testnets/raw/master/shared/prater/genesis.ssz")
```

Comment:
Sorry to keep moving the goal posts, it occurs to me that we shouldn't assume the state in master won't change.  Pointing to the specific commit would solve that.
```suggestion
              "https://github.com/eth2-clients/eth2-testnets/raw/192c1b48ea5ff4adb4e6ef7d2a9e5f82fb5ffd72/shared/prater/genesis.ssz")
```